--- llvm-config.orig	Fri Apr 30 22:39:09 2010
+++ llvm-config	Wed May  5 16:12:29 2010
@@ -18,7 +18,6 @@
 use 5.006;
 use strict;
 use warnings;
-use Cwd 'abs_path';
 
 #---- begin autoconf values ----
 my $PACKAGE_NAME        = q{llvm};
@@ -52,7 +51,12 @@
 # locations of some files.
 
 # Convert the current executable name into its directory (e.g. ".").
-my ($RUN_DIR) = ($0 =~ /^(.*)\/.*$/);
+# JeF Hardcode path as c:/llvm/bin because it is the only form that works
+# both inside Qt creator and in a bash window.
+my $RUN_DIR = q(c:/llvm/bin); 
+sub abs_path {
+    return $_[0];
+}
 
 # Turn the directory into an absolute directory on the file system, also pop up
 # from "bin" into the build or prefix dir.
