--- RWMutex.cpp.old	Sun May  2 01:22:20 2010
+++ RWMutex.cpp	Fri Apr 30 23:32:26 2010
@@ -84,11 +84,11 @@
 
     // Initialize the rwlock
     errorcode = pthread_rwlock_init(rwlock, &attr);
-    assert(errorcode == 0);
+    //assert(errorcode == 0);
 
     // Destroy the attributes
     errorcode = pthread_rwlockattr_destroy(&attr);
-    assert(errorcode == 0);
+    //assert(errorcode == 0);
 
     // Assign the data member
     data_ = rwlock;
